app/bundles/LeadBundle/Entity/CompanyRepository.php

```diff
@@ -417,14 +418,21 @@ public function getAjaxSimpleList(CompositeExpression $expr = null, array $param
             }
         }
 
-        $q->select($prefix.$valueColumn.' as value,
-        case
-        when (comp.companycountry is not null and comp.companycity is not null) then concat(comp.companyname, \' <small>\', companycity,\', \', companycountry, \'</small>\')
-        when (comp.companycountry is not null) then concat(comp.companyname, \' <small>\', comp.companycountry, \'</small>\')
-        when (comp.companycity is not null) then concat(comp.companyname, \' <small>\', comp.companycity, \'</small>\')
-        else comp.companyname
-        end
-        as label')
+        if (!$onlyNames) {
+            $labelExpression = '
+            case
+            when (comp.companycountry is not null and comp.companycity is not null) then concat(comp.companyname, \' <small>\', companycity,\', \', companycountry, \'</small>\')
+            when (comp.companycountry is not null) then concat(comp.companyname, \' <small>\', comp.companycountry, \'</small>\')
+            when (comp.companycity is not null) then concat(comp.companyname, \' <small>\', comp.companycity, \'</small>\')
+            else comp.companyname
+            end
+            as label';
+        }
+        else {
```

Comment:
Consistency with the rest of the code base
```suggestion
        } else {
```